app/src/dbRequest.js

```diff
@@ -0,0 +1,57 @@
+//Task Requests
+export const getTasks = async (user) => {
+  const response = await fetch("/api/tasks", {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+      "timedown-user": user,
+    },
+  });
+  return response.json();
+};
+
+export const addTask = async (
+  userId,
+  dueDate,
+  estTime,
+  summary,
+  description,
+) => {
+  const response = await fetch("/api/tasks", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({ userId, dueDate, estTime, summary, description }),
+  });
+
+  return getTasks(userId);
```

Comment:
Hmmmm, if I were a user of the `addTask` function, I'm not sure I would expect it to make this 2nd API request.  I'm guessing you're making this call because you're using its response to show *all* the user's tasks, including the one you just added?  If so, there may be other ways to accomplish the same task.  Worst-case scenario, from an architectural standpoint I'd advocate for leaving this 2nd call up to the code which calls `addTask`.